lib/rubill/session.rb

```diff
@@ -67,8 +67,19 @@ def _post(url, data, retries=0)
     end
 
     def self._post(url, options)
-      body = post(url, body: options, headers: default_headers).body
-      result = JSON.parse(body, symbolize_names: true)
+      if options.key?(:fileName)
+        file = StringIO.new(options.delete(:content))
+      end
+
+      post_options = {
+        body: options,
+        headers: default_headers,
+      }
+
+      post_options[:file] = file if file
```

Comment:
Correct me if I'm wrong, but won't this throw a `NameError` if file has not been defined on line 71?